src/tic_one.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _tic_one{T<:AbstractFloat}(zmin::T, pixx::T, incr::T, ra::Bool)
+    if ra
+        mul = T(4)
```

Comment:
In this case you don't need to cast the type of the constant because `mul` is type-stable (in either case is always an `Int`, instead in `tics` function `incr` could have been both an `Int` and a `Rational`) and its value will be automatically converted during computation ;-)